feat(lua): add :verbose support for lua config (#15079)
neovim/neovim: runtime/doc/various.txt

							*:verbose-cmd*
When 'verbose' is non-zero, listing the value of a Vim option or a key map or
an abbreviation or a user-defined function or a command or a highlight group
or an autocommand will also display where it was last defined. If they were

> defined through lua api they will only be located if verbose level was

Reviewer: - always capitalize lua in docs and messages, unless it is referring to a filename or symbol.
- Acronyms like "API" are all caps unless... they are filenames or symbols. (btw: "id" is not an acronym).
- "the Lua API" is generally not mentioned. It's just "Lua".

 
```suggestion
defined in Lua they will only be located if 'verbose' is set.
```

> greater then 0 when that map/abbrev/option/function.. was defined. So

Reviewer: delete this line

Author: > If they were
defined through lua api they will only be located if verbose level was
greater then 0 when that map/abbrev/option/function.. was defined

This one ?

Reviewer: no. only the line i commented on. I gave the suggested form above.
